### DISCUSSION TITLE
Master theming reference for LXQt themes?

### DISCUSSION THREAD

**Discussion by AzumaHazuki** [lxqt/lxqt#2347] [Q&A]
Is there a document anywhere listing all the items we can theme with .qss stylesheets and all the states they can be in?

**Comment by tsujan** [comment]
[Qt doc.](https://doc.qt.io/qt-6/stylesheet-reference.html)

**Reply by AzumaHazuki** [reply]
Yes, I am well aware of the general .qss styling reference and box model documentation. I am asking specifically about those widgets LXQt has created/extended from the base Qt set. And I think you knew that.
